specs/TLA+/LeaderFollower/AbstractVeniceLeaderFollower.tla

```diff
@@ -10,10 +10,10 @@
 (***************************************************************************)
 
 EXTENDS Integers, Sequences
-CONSTANTS KEYS, VALUES, N_NODES, MAX_WRITES
-VARIABLE realTimeTopic, versionTopic, nodes
+CONSTANTS KEYS, VALUES, N_NODES, MAX_WRITES, MAX_VERSION_PUSHES
```

Comment:
Consider a [state constraint](https://github.com/tlaplus-workshops/ewd998/blob/17505f391d794a3d2e2f25c529877dd0c602f87a/MCAsyncTerminationDetection.tla#L9-L19) (`CONSTRAINT`) instead of hardcoded bounds to constrain the state space.

Reply:
Good idea!  Added.